src/rutujaGDec19/Assignment3/Question1.java

```diff
@@ -0,0 +1,41 @@
+package rutujaGDec19.Assignment3;
+/*
+ * To write the positive and negative numbers from an array 
+ * to print their sum
+ * */
+import java.util.Scanner;
+public class Question1 {
+	static void Check(int[] a) {
+		int psum = 0;
+		int nsum = 0;
+		    for(int i = 0; i < a.length; i++)
+		    {
+		      if(a[i] >= 0)
+		      {
+		        System.out.println("Positive Number:"+a[i]);
```

Comment:
i want separate array of positive number and negative, my goal is to achieve array of positive and negative